README.md

```diff
@@ -7,7 +7,7 @@
 
 # Puppet VSCode [Website](https://puppet-vscode.github.io)
 
-We are proud to announce the new Puppet VSCode website [https://puppet-vscode.github.io/](https://puppet-vscode.github.io/)! This website contains all of the documenation previously held in this README, simplified and organinzed. There is also new content that shows off all the current features in greater detail, as well as new ways to use the Puppet VSCode Extension with other technologies like the [Microsoft Remote Development Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack). This README retains some quick information on how to get started and where to get help if things go wrong.
```

Comment:
Didn't want to modify the formatting, but unfortunately this is all on one line.

Changes `documenation` to `documentation`